src/NpgsqlFSharpAnalyzer.Core/SqlAnalysis.fs

```diff
@@ -511,15 +514,7 @@ module SqlAnalysis =
                                     ()
                                 else if not requiredParam.DataType.IsArray then 
                                     match requiredParam.DataType.Name with
-                                    | "bit" ->
```

Comment:
@falfaddaghi  this the only part that needs some thought to make it work for both postgres and sql server.